meshroom/ui/qml/Controls/ExifOrientedViewer.qml

```diff
@@ -0,0 +1,58 @@
+import QtQuick 2.11
+
+/**
+ * Loader with a predefined transform to orient its content according to the provided Exif tag.
+ * Useful when displaying images and overlayed information in the Viewer2D.
+ * 
+ * Usage:
+ * - set the orientationTag property to specify Exif orientation tag.
+ * - set the xOrigin/yOrigin properties to specify the transform origin.
+ */
+Loader {
+	property var orientationTag: undefined
+
+	property real xOrigin: 0
+	property real yOrigin: 0
+
+    transform: [
```

Comment:
There is an indentation issue here, there are more spaces for the properties than for the `transform`.